.gitlab-ci.yml

```diff
@@ -187,6 +187,54 @@ spellcheck:
         cargo spellcheck check -vvv --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1 examples/delegator/${contract}/;
       done
 
+codecov:
+  stage:                           workspace
+  <<:                              *docker-env
+  <<:                              *test-refs
+  needs:
+    - job:                         check-std
+      artifacts:                   false
+  variables:
+    # For codecov it's sufficient to run the fuzz tests only once.
+    QUICKCHECK_TESTS:              1
+    # Now we're using the MIR source-based Rust test coverage, the setup assumes that default
+    # toolchain is nightly and both `grcov` and `rustup component add llvm-tools-preview` are
+    # installed on it.
+    RUSTFLAGS:                     "-Zinstrument-coverage"
```

Comment:
Why did we have all the other `RUSTFLAGS` before and why is it fine to get rid of them now?

Reply:
> Why did we have all the other RUSTFLAGS

https://github.com/paritytech/ink/pull/857/files#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7L199

> why is it fine to get rid of them now

some of them were there just in order to make `grcov` at least work, others were to fix the coverage pitfalls. Details above.